price-feeder/oracle/provider/provider.go

```diff
@@ -101,8 +109,8 @@ func PastUnixTime(t time.Duration) int64 {
 	return time.Now().Add(t*-1).Unix() * int64(time.Second/time.Millisecond)
 }
 
-// mapPairsToSlice returns the map of currency pairs as slice.
-func mapPairsToSlice(mapPairs map[string]types.CurrencyPair) []types.CurrencyPair {
+// MapPairsToSlice returns the map of currency pairs as slice.
+func MapPairsToSlice(mapPairs map[string]types.CurrencyPair) []types.CurrencyPair {
```

Comment:
This should probably be in `types/currency.go`

Reply:
done at https://github.com/umee-network/umee/pull/649/commits/6b38bd0290cadf66db4c737d03c0c364cccca746